src/components/Inputs/EmailInput/EmailInput.js

```diff
@@ -44,20 +25,16 @@ const PrivateEmailInput = (props) => {
         initialValue={value}
         optional={optional}
         type="email"
+        disabled={disabled}
         data-tid={restProps['data-tid']}
         placeholder={placeholder}
-        currentValue={currentValue}
         validator={EmailFormatValidator}
-        currentError={currentError}
-        formTouched={formTouched}
-        setFieldTouched={restProps.setFieldTouched}
       />
-      {getError(currentError, touched)}
```

Comment:
Have you been able to configure it such that the TextInput is essentially kicking off the whole validation and `getError()` calls? If so, this is good in terms of removing `getError`

Are you passing through `...rest` outside of the diff? I think we need currentValue, currentError, etc.

Reply:
Yeah, I think you need to let `...restProps` pass through. Not sure if that has anything to do with the fields not being validated